src/com/facebook/buck/apple/AppleBundle.java

```diff
@@ -409,7 +421,7 @@ public boolean isLegacyWatchApp() {
     }
 
     if (!frameworks.isEmpty()) {
-      Path frameworksDestinationPath = bundleRoot.resolve(this.destinations.getFrameworksPath());
```

Comment:
seems legit, ref. to your comment here https://github.com/AquaGeek/buck/pull/1/files/cab05071d50ac59892bac30c42d14393c11b8c0e#r80579009, how do you think about creating a step inside a `AppleBundle` and get it in here? just my 2cents.